src/libs/actions/Task.js

```diff
@@ -13,6 +13,16 @@ import * as UserUtils from '../UserUtils';
 import * as PersonalDetailsUtils from '../PersonalDetailsUtils';
 import * as ReportActionsUtils from '../ReportActionsUtils';
 
+let currentUserEmail = '';
+let currentUserAccountID = 0;
```

Comment:
NAB, but I don't think you have to set initial values. You could just do: 
```suggestion
let currentUserEmail;
let currentUserAccountID;
```

Reply:
oh i think you're def right 👍